src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/HomeTestResultCell.swift

```diff
@@ -31,99 +31,9 @@ class HomeTestResultCell: HomeCardCollectionViewCell {
 	@IBOutlet weak var image: UIImageView!
 	@IBOutlet weak var body: UILabel!
 	@IBOutlet weak var button: ENAButton!
-
-	weak var delegate: TestResultCollectionViewCellDelegate?
+	weak var delegate: HomeTestResultCellDelegate?
 
```

Comment:
IB is crashed when it tries to open .xib 

![image](https://user-images.githubusercontent.com/61236601/83555371-79371680-a50e-11ea-9c54-97ca7e7944c2.png)

Reply:
had the same issue from time to time, i thought it was only my machine but i will see if i can find a reason why xcode does not like the xib

Reply:
I think, the reason is in some ibdesignable view in this .xib

Reply:
i could not figure out what causes the issue. i can open and edit it normally and sometimes it just crashes. :(

Reply:
No crash on my end… not sure what to do. The change @johannesrohwer did was not that big and does not look suspicious.